rts/lra/lra-coordinator-jar/pom.xml

```diff
@@ -81,9 +81,9 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>io.smallrye</groupId>
-            <artifactId>smallrye-open-api</artifactId>
-            <version>1.1.19</version>
+            <groupId>org.eclipse.microprofile.openapi</groupId>
+            <artifactId>microprofile-openapi-api</artifactId>
+            <version>1.1.2</version>
```

Comment:
@xstefank just to understand, should this really be in `compile` scope? as this is expected to be deployed to WildFly that should be rather the WFLY module dependency which will be used not the one compiled in, right?
and that side note, the version should be probably configurable?

Reply:
it can. Sorry, I forgot about this comment.